crates/rust-analyzer/tests/rust-analyzer/main.rs

```diff
@@ -891,7 +891,9 @@ fn test_shows_codelens() {
     let tmp_dir = TestDir::new();
     let tmp_dir_path = tmp_dir.path().to_owned();
     let tmp_dir_str = tmp_dir_path.to_str().unwrap();
-    let lib_path = PathBuf::from(format!("file://{}/src/lib.rs", tmp_dir_str).replace("\\", "/"));
+    let lib_path = PathBuf::from(
+        format!("file://{}/src/lib.rs", tmp_dir_str).replace("\\", "/").replace("C:/", "/C:/"),
```

Comment:
The user may not necessarily use their `C:\` drive to store their checkout dir. It may be any other drive.

Reply:
In Zulip discussion it was concluded that this test should be removed after CodeLens refactoring (WIP) and replaced with a better one.